src/interfaces/python/swig_typemaps.i

```diff
@@ -1415,7 +1415,7 @@ for getter in _GETTERS:
     _rename_python_function(_shogun.SGObject, getter, _private_getter)
     _internal_getter_methods.append(_shogun.SGObject.__dict__[_private_getter])
 
-def _internal_get_param(self, name):
+def _internal_get_param(self, name, *args):
```

Comment:
@gf712 I added this to account for the index arguments. You agree with it?

Reply:
seems fine to me!